tests/integration-test.sh

```diff
@@ -357,7 +347,7 @@ log "Node 5 should now have a ticket"
 result=$(api_get_ticket_statistics "${api5}" "\"winProportion\":1")
 log "-- ${result}"
 
-for i in `seq 1 100`; do
+for i in `seq 1 20`; do
```

Comment:
Why lowering from a 100?

Reply:
I was attempting to lower the number of messages to lower the number of tickets and not hit the automatic aggregation.